### DISCUSSION TITLE
Installation guide

### DISCUSSION THREAD

**Discussion by Zunnen** [openai/whisper#334] [Q&A]
Hey, I'm struggeling to actually install whisper and use it. If someone has the time, could you please give me a short step by step guide on how to install it/which programs i need?
Thanks a lot in advance, I've never actaully donwloaded one befor :)

**Comment by FurkanGozukara** [comment]
I have a video that explains how to use whisper it may help. i install on windows from scratch and run it

https://www.youtube.com/watch?v=msj3wuYf3d8
